CandidateWords.java

```diff
@@ -181,12 +189,12 @@ private static void getInput(ArrayList<String> words)
           if (removed)
           {
             removed = false;
-            break;
+            j--;
```

Comment:
This is the main fix for the words not matching the yellow letter condition not being properly removed. Previously the code exited after the first yellow condition removed a word. @toddcscar-prime